honeybee_vtk/assistant.py

```diff
@@ -37,6 +37,16 @@ def __init__(self, background_color: Tuple[int, int, int], camera: Camera,
         self._camera = camera
         self._legend_params = legend_parameters
         self._text_actor = text_actor
+        self._vtk_camera = None
+        self._active_camera = None
+
+    @property
+    def vtk_camera(self) -> vtk.vtkCamera:
+        return self._vtk_camera
+
+    @vtk_camera.setter
+    def vtk_camera(self, vtk_camera):
+        self._vtk_camera = vtk_camera
```

Comment:
We probably need a check here to ensure it is the correct type.

Reply:
I did that. This is one of those things that we go back and forth on. I tend to write in what is called defensive programming style where you set up checks for such types. In many of the earlier comments, you have not liked it. I would agree that every situation is different and we cannot have a blanket rule though.